test/parallel/test-quic-simple.js

```diff
@@ -15,7 +15,7 @@ const createSocket = require('quic');
 
 const socket = createSocket({ type: 'udp4', port: 1234 });
 
-socket.listen({ key, cert, ca });
+socket.listen({ key, cert, /*ca*/ });
```

Comment:
For some reason I've yet to determine, adding the `ca` in the to secure context causes a handshake failure in the ngtcp2 example client. It's not yet clear if that's a bug on our end or theirs.